quickstarts/nrm4sap/config.yml

```diff
@@ -0,0 +1,61 @@
+# Long-form description of the quickstart (required)
+description: |
+  ## Why New Relic Monitoring for SAP® Solutions 
+
+  With New Relic Monitoring for SAP® Solutions, you can inspect all of your SAP and non-SAP systems through one centralized view of your infrastructure, application, and business processes.
+  -- Our solution is designed to not impact performance of the SAP production servers, and instead concentrates the monitoring load on a central, non-production server.
+  -- The solution is SAP Certified, and installs quickly and conveniently, allowing for easy installation with as little as 1-2 hour required.
+  -- Once our SAP connector is installed, all telemetry data from SAP is conveniently available via out-of-the-box dashboards, system views, business process views, and other powerful visualizations in the New Relic platform.
+  -- Monitor SAP application performance and overall health data in one place. Telemetry metrics, events, logs, and traces all come together in a central view within a single UI.
+  -- View your data in relation to your business processes. Understand your infrastructure and application data from the viewpoint of your corresponding business processes, such as order-to-cash and procure-to-pay.
+  -- Spot problems quickly with visualizations. By querying and viewing relevant data in customizable charts and dashboards, you can identify root causes and resolve problems faster.
+  -- Prevent problems before they occur. Use our advanced alerting functionality to be notified about problems before they impact customers.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  A SAP-Certified integration that brings together all your SAP and non-SAP telemetry–events, metrics, logs and traces into a unified data platform, for a single view of infrastructure, application and business processes.
+
+# Support level: New Relic | Verified | Community (required)
+level: New Relic 
+
+# Authors of the quickstart (required)
+authors:
+  - New Relic 
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - sap 
+  - hana
+  - integration 
+  - idoc
+  - rfc
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+alertPolicies:
+  - nrm4sap 
+dashboards:
+  - nrm4sap 
+  - nrm4sap-datacollect
+  - nrm4sap-datasize
+# Documentation references
+documentation:
+  - name: Installation docs
+    url: https://drive.google.com/file/d/1PeCLM3vQxc-lTxt5R8ZZ6kx9kkfJEu0Y/view?usp=sharing 
+    description: Installation Guide for New Relic Monitoring for SAP® Solutions 
+  - name: New Relic Monitoring for SAP® Solutions 
+    description: New Relic Monitoring for SAP® Solutions feature overview 
+    url: https://newrelic.com/solutions/sap
+  - name: Administration Guide 
+    url: https://drive.google.com/file/d/1fX37P3z9DH4ZgQO9xpJjzP2ZQ3ReA7Dl/view?usp=sharing 
```

Comment:
Here as well

Reply:
The baseline alerts conditions have been updated to include "operator"

Reply:
@edyu002 - @seemantk will be reaching out in regards to this :)